src/tactic/suggest.lean

```diff
@@ -518,8 +521,10 @@ nat.one_pos
   descr := "Use `library_search` to complete the goal.",
   action := λ _, do
     script ← library_search,
-    -- Is there a better API for dropping the 'exact ' prefix on this string?
-    return [((script.mk_iterator.remove 6).to_string, "by library_search")] }
+    -- Is there a better API for dropping the 'Try this: exact ' prefix on this string?
+    let s := script.to_list,
+    return [(((s.get_rest "Try this: exact ".to_list).get_or_else s).as_string,
+            "by library_search")] }
```

Comment:
Tempting to add `get_rest` to the string API implemented as `(s1.to_list.get_rest s2.to_list).map list.as_string` or similar, which would make this

```suggestion
    return [((script.get_rest "Try this: exact ").get_or_else s, "by library_search")] }
```

Reply:
Wow, `string.get_rest` already exists! Not sure how I missed that...